src/components/sidebar.tsx

```diff
@@ -1,24 +1,85 @@
-import { Link } from "react-router-dom";
+import { NavLink } from "react-router-dom";
+import logo from "../assets/images/logo.png";
+
+import user from "../assets/icons/user.svg";
+import projects from "../assets/icons/projects.svg";
+import schedule from "../assets/icons/schedule.svg";
+import chat from "../assets/icons/chat.svg";
```

Comment:
I suggest that you replace relative paths with absolute using "@/assets/images/...", because if we move the file to another place OR use the same path into another file, we can import it easier

Reply:
I did your excelent suggestion, thank you, i had to change de jsconfig file for that